setup.py

```diff
@@ -49,20 +49,17 @@ def read(fname):
             "py3-cmd = py3status.command:send_command",
         ]
     },
+    python_requires=">=3.5",
     classifiers=[
         "License :: OSI Approved :: BSD License",
         "Operating System :: POSIX :: Linux",
         "Programming Language :: Python",
-        "Programming Language :: Python :: 2.6",
-        "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3",
```

Comment:
I guess we can remove this, no?

Reply:
It can, but it's very commonly used to mean _any_ 3.x is supported:

https://pypi.org/search/?c=Programming+Language+%3A%3A+Python+%3A%3A+3

There's another classifier for _exactly_ 3.0:

https://pypi.org/search/?c=Programming+Language+%3A%3A+Python+%3A%3A+3.0

Reply:
When this first iteration of cleanup will have taken place, we will drop support for 3.5 and thus will need to remove this

But for now, that's good yep